optimade/server/tests/test_server.py

```diff
@@ -17,8 +16,14 @@
     InfoResponse,
 )
 
+from optimade.server.main import app
+from optimade.server.routers import info, references, structures
+
 # need to explicitly set base_url, as the default "http://testserver"
 # does not validate as pydantic UrlStr model
+app.include_router(info.router)
+app.include_router(references.router)
+app.include_router(structures.router)
 CLIENT = TestClient(app, base_url="http://example.org/optimade")
```

Comment:
Could you just move the comment down so it still corresponds to the TestClient definition please?